assets/queries/k8s/limit_capabilities_from_the_container/query.rego

```diff
@@ -0,0 +1,95 @@
+package Cx
+
+CxPolicy [ result ] {
+   spec := input.document[i].spec 
+   drop := spec.containers[c].securityContext.capabilities.drop
+   not drop[0] == "ALL" 
```

Comment:
this should validate differently.

What if we have:
 
drop: ["NET_ADMIN", "SYS_TIME", "ALL"]